.github/workflows/dependabot-create-issue.yml

```diff
@@ -0,0 +1,31 @@
+name: Create issue for dependabot pr
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+    types: [opened, reopened]
+
+
+jobs:
+    create_issue:
+        runs-on: ubuntu-latest
+        env:
+            GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+        steps:
+          - uses: actions/checkout@v2
+            if: github.event.pull_request.user.login  == 'dependabot[bot]'
+
+          - name: Open issue if Dependabot PR
+            if: github.event.pull_request.user.login  == 'dependabot[bot]'
+            env:
+              pr_title: ${{github.event.pull_request.title}}
+              pr_number: ${{github.event.pull_request.number}}
+              pr_url: ${{github.event.pull_request.url}}
+            run: |
+              project="Data Catalogue"
+              label="Dependabot Issue"
+              title="Dependabot PR $pr_title opened"
```

Comment:
Can we name this something like "Review dependabot PR: $pr_title" so it reflects the task that needs doing?

Also, where's the emoji? 🧌

Reply:
Updated and Emoji Added :-)